config/initializers/csv_renderer.rb

```diff
@@ -1,27 +1,29 @@
 # Currently only used for rendering collections of stories, so make an
 # assumption that that is what we get passed.
 ActionController::Renderers.add :csv do |stories, options|
-
+  number_of_extra_columns = {}
+  number_of_extra_columns[:notes] = stories.map{|story| story.notes.length}.max
```

Comment:
Rubocop isn't set for this project?

It should tell you to put a single space before/after curly brackets...